rex-macros/src/tracepoint.rs

```diff
@@ -1,13 +1,9 @@
 use proc_macro2::TokenStream;
-use proc_macro_error::OptionExt;
-use quote::{format_ident, quote};
-use syn::{parse2, ItemFn, Result};
-
-use crate::args::parse_string_args;
+use proc_macro_error::{abort_call_site, OptionExt};
+use quote::{format_ident, quote, ToTokens};
+use syn::{parse2, FnArg, Ident, ItemFn, Result, Type};
 
 pub(crate) struct TracePoint {
-    tp_type: Option<String>,
-    name: Option<String>,
     item: ItemFn,
 }
 
```

Comment:
Just one more thing, could you help remove the comments here, since current style is not aya style.

Reply:
We could more this PR and remove this in another commit